docs/snippets/fork.mdx

```diff
@@ -0,0 +1,6 @@
+All contributions to Prefect need to start on [a fork of the repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo).
+Once you have successfully forked [the Prefect repo](https://github.com/PrefectHQ/prefect), create a branch with an informative name:
+```
+git checkout -b fix-for-issue-NUM
+```
+After committing your changes to this branch, you can then open a pull request from your fork that we will review with you.
```

Comment:
```suggestion
After committing your changes to this branch, you can then open a pull request from your fork that we will review.
```

Reply:
The "with you" was intentional to highlight that review is a two-way conversation.